experimental/packages/exporter-trace-otlp-http/test/browser/CollectorTraceExporter.test.ts

```diff
@@ -103,35 +103,36 @@ describe('OTLPTraceExporter - web', () => {
         });
 
         setTimeout(async () => {
-          const args = stubBeacon.args[0];
-          const url = args[0];
-          const blob: Blob = args[1];
-          const body = await blob.text();
-          const json = JSON.parse(
-            body
-          ) as otlpTypes.opentelemetryProto.collector.trace.v1.ExportTraceServiceRequest;
-          const span1 =
-            json.resourceSpans[0].instrumentationLibrarySpans[0].spans[0];
-
-          assert.ok(typeof span1 !== 'undefined', "span doesn't exist");
-          if (span1) {
-            ensureSpanIsCorrect(span1);
-          }
-
-          const resource = json.resourceSpans[0].resource;
-          assert.ok(typeof resource !== 'undefined', "resource doesn't exist");
-          if (resource) {
+          try {
+            const args = stubBeacon.args[0];
+            const url = args[0];
+            const blob: Blob = args[1];
+            const body = await blob.text();
+            const json = JSON.parse(
+              body
+            ) as IExportTraceServiceRequest;
+            const span1 =
+              json.resourceSpans?.[0].scopeSpans?.[0].spans?.[0];
+
+            assert.ok(typeof span1 !== 'undefined', "span doesn't exist");
+            if (span1) {
```

Comment:
This if-branch can be removed too.

Reply:
Agreed. :slightly_smiling_face: This pattern seems to already exist in a bunch of tests already actually. :thinking: Removed in [35d9762](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/35d976293aff242f9e0664e543f6a6b3dca7059e)

Reply:
I found the rest of these and fixed them as well. ([47aa7e4](https://github.com/open-telemetry/opentelemetry-js/pull/2929/commits/47aa7e468e2e6d3cf7e235869d9a50d16b37ee34)) :slightly_smiling_face: